cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/index.js

```diff
@@ -156,7 +175,9 @@ class RunsGraphView extends Component {
 
           <div
             className="navigation arrow-left"
-            onClick={prev}
+            disabled={this.props.is7DaysAgo}
```

Comment:
Shouldn't do this. Add a class for the disabled styling. `disabled` is not a valid HTML semantics for `div`